### PR TITLE
Design Teardown: The Smashing Magazine Website

### PR DESCRIPTION
**Pull request by DeliceLydia**
In this project, we created a Heatmap of the Smashing Magazine Website using HTML and CSS.

Additional Information:

- Setup project dependencies like GitHub Actions and Linters via Microverse Linters Config.

- Used semantic tags and others were necessary.

- Used different colors of grey to show the structure of the webpage.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+
+#featured-article2 {
+  background-color: #b6afaf;
+}
+
+#featured-article3 {
+  background-color: #867e7e;
+}
+
+#featured-article4 {
```
Comment (andresporras3423) [inline css/style.css]: Don't overuse ID selectors

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+}
+
+.icon-image {
+  display: flex;
+  margin-left: 200px;
+}
+
+.icon-image button:hover {
+  background-color: #fff;
+}
```
Comment (andresporras3423) [inline css/style.css]: Group with selector on line 114

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Delice Ingabire** 
Good job with your first submission for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] Don't overuse ID Selectors
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.
- [x]  All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of gray), this rule also applies to images:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello @DeliceLydia :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**